region/maridia/inner-pink.json

```diff
@@ -5305,7 +5305,7 @@
                     "canIframeSpikeJump",
                     {"spikeHits": 3},
```

Comment:
I do a controlled neutral boost on the far left spike for positioning, maybe you can uninteruptable animation frames to avoid that.